src/main/java/org/qualityannotate/quality/sonarqube/SonarqubeQualityTool.java

```diff
@@ -22,50 +21,58 @@ public class SonarqubeQualityTool implements QualityTool {
     public SonarqubeQualityTool(SonarqubeConfig config) {
         this.config = config;
         client = QuarkusRestClientBuilder.newBuilder()
-                                         .baseUri(URI.create(config.url()))
-                                         .register(new BasicAuthRequestFilter("", config.token()))
-                                         .build(SonarqubeApiClient.class);
+                .baseUri(URI.create(config.url()))
+                .register(new BasicAuthRequestFilter(config.token()))
+                .build(SonarqubeApiClient.class);
     }
 
     List<Issue> getIssues() {
-        IssueSearch issuesSearch =
-                client.getIssuesSearch(config.project(), config.pullRequest(), null, null, null, null, null, null, null,
-                        null, null);
+        IssueSearch issuesSearch = client.getIssuesSearch(config.project(), config.pullRequest(), null);
         // TODO issueSearch.facets can be used for globalMetrics - but maybe doesn't matter
         List<Issue> issues = new ArrayList<>();
         for (SqIssue sqIssue : issuesSearch.issues()) {
-            issues.add(new Issue(sqIssue.getPath(config.project()), sqIssue.lineNumber(), sqIssue.message(),
-                    sqIssue.severity(), null));
+            issues.add(new Issue(
+                    sqIssue.getPath(config.project()),
+                    sqIssue.textRange().startLine(),
+                    sqIssue.message(),
```

Comment:
🛑 Refactor this method to reduce its Cognitive Complexity from 18 to the 15 allowed. [details](https://sonarcloud.io/scoding_rules?open=java:S3776&rule_key=java:S3776)